src/ResourceManager/Automation/Commands.Automation/Common/AutomationPSClientDSC.cs

```diff
@@ -790,50 +795,138 @@ public void DeleteDscNode(string resourceGroupName, string automationAccountName
                 azureVmResourceGroup = resourceGroupName;
             }
 
-            var deploymentName = Guid.NewGuid().ToString();
-
             // get the endpoint and keys
             Model.AgentRegistration agentRegistrationInfo = this.GetAgentRegistration(
                 resourceGroupName,
                 automationAccountName);
 
-            // prepare the parameters to be used in New-AzureRmResourceGroupDeployment cmdlet
-            Hashtable templateParameters = new Hashtable();
-            templateParameters.Add("vmName", azureVMName);
-            templateParameters.Add("location", location);
-            templateParameters.Add("modulesUrl", Constants.ModulesUrl);
-            templateParameters.Add("configurationFunction", Constants.ConfigurationFunction);
-            templateParameters.Add("registrationUrl", agentRegistrationInfo.Endpoint);
-            templateParameters.Add("registrationKey", agentRegistrationInfo.PrimaryKey);
-            templateParameters.Add("nodeConfigurationName", nodeconfigurationName);
-            templateParameters.Add("configurationMode", configurationMode);
-            templateParameters.Add("configurationModeFrequencyMins", configurationModeFrequencyMins);
-            templateParameters.Add("refreshFrequencyMins", refreshFrequencyMins);
-            templateParameters.Add("rebootNodeIfNeeded", rebootFlag);
-            templateParameters.Add("actionAfterReboot", actionAfterReboot);
-            templateParameters.Add("allowModuleOverwrite", moduleOverwriteFlag);
-            templateParameters.Add("timestamp", DateTimeOffset.UtcNow.ToString("o"));
-
             // invoke the New-AzureRmResourceGroupDeployment cmdlet
             using (var pipe = System.Management.Automation.PowerShell.Create(RunspaceMode.NewRunspace))
```

Comment:
Please don't create a PowerShell runspace, as it is not used with your new changes.